test/uk/gov/hmrc/gform/auth/AuthServiceSpec.scala

```diff
@@ -49,6 +49,7 @@ class AuthServiceSpec extends ExampleData with Spec with TableDrivenPropertyChec
     `auth-module` = null,
     /*we can't override list in app-config-base:*/
     contentTypesSeparatedByPipe = "csv|txt",
+    restrictedFileExtensionsSeparatedByPipe = "jfif",
```

Comment:
This `*SeparatedByPipe` is such a pain and error prone. Let's not repeat it again. Can't we use array of Strings instead?